salt/utils/roster.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+
+'''
+Provide range capabilities to rosters.
+'''
+
```

Comment:
Can you add this import here:
```
from __future__ import absolute_import
```

That will take care of the "Python 3 incompatible" lint error. There are a few more lint errors defined here:
https://jenkins.saltstack.com/job/PR/job/salt-pr-lint-n/16712/violations/file/salt/utils/roster.py/

Can you clean those up? :)

Reply:
Addressed, thanks